.eslintrc.yml

```diff
@@ -0,0 +1,27 @@
+extends:
+  - defaults/configurations/airbnb/es5
+rules:
+  no-empty-label: off
+  space-after-keywords: off
+  space-return-throw-case: off
+  no-labels: error
+  keyword-spacing: error
```

Comment:
The five lines above disable some deprecated rules in the `defaults/configurations/airbnb/es5` config, and replace them with equivalent ones.

Once `eslint-config-defaults` 10.0.0 is released, these lines can be removed.